src/hangedMan/domain/LeftGallows.java

```diff
@@ -1,3 +1,7 @@
+package hangedMan.domain;
```

Comment:
Please refer to this article https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html about naming conventions for package names. The usage of camel case in package name is not a good practice. Since you do not own any domain and this is purely educational project, I suggest you use your github username or your initials (or something similar) as prefix, i.e. klarasindelarova.hangedman or ks.hangedman or simmilar.

Also since this project is small, it is very difficult and also unconvenient to categorize every class into apropriate package, so I suggest to use only single level, and put all classes to the same package.

Reply:
Thank you for the resource. I specifically checked the course material because of the camel case and they showed us this one: package flightControl.domain; However, I changed the name of the package according to the new instructions.